<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Report</class>
 <widget class="QDialog" name="Report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>610</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_updated">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>XXXX productos actualizados</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Actualizacion completada</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_notfound">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Los siguientes productos no han encontrado (codigo inexistente)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_notmatched">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Los siguientes productos no coinciden con la descripcion (codigo dado de alta)</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lst_notmatched">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>671</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lst_notfound">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>671</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Report</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Report</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
